src/Simplex/Chat/Protocol.hs

```diff
@@ -112,6 +113,14 @@ agentToChatVersion v
   | v < pqdrSMPAgentVersion = initialChatVersion
   | otherwise = pqEncryptionCompressionVersion
 
+data ConnectionEntityId
```

Comment:
probably would make more sense to have this together with ConnectionEntity somewhere in Types, but since ConnectionEntity is currently here, didn't move